man/systemd.network.xml

```diff
@@ -1631,6 +1631,13 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>Label=</varname></term>
+          <listitem>
+            <para>An address label.</para>
```

Comment:
I know that you just copied the explanation for `Label=` in `[Address]` section. But could you slightly extend this? At least, please mention the default value? How about the following:
```
Specifies the label for the acquired DHCPv4 address. Defaults to unset.
```

Also, please see the comments in `<!-- -->`. This settings should be moved just above `UseDNS=`, as this setting is for how to handle the acquired lease.